SPARKBOX-SETUP.md

```diff
@@ -0,0 +1,99 @@
+# Sparkbox - Craft
+
+A fork of Craft CMS - https://github.com/craftcms/craft
+
+## Table of Contents
+
+* [Install and Configure Craft](#install-and-configure-craft)
+    * [Install and Configure Apache](#install-and-configure-apache)
+    * [Installing MySQL](#installing-mysql)
+    * [Install Sequel Pro](#install-sequel-pro)
+    * [Sparkbox Craft Install](#sparkbox-craft-install)
+    * [Create Craft Project Database](#create-craft-project-database)
+    * [Run Craft](#run-craft)
+* [Common Issues](#common-issues)
+    * [Fields Not Saving](#fields-not-saving)
+        
+## Install and Configure Craft
+
+### Install and Configure Apache
+
+Follow the [Apache at Sparkbox](https://github.com/sparkbox/standard/tree/master/software/apache) guide in the Standard to install and configure Apache on your machine.
+
+### Installing MySQL
+
+We will not be using homebrew for mysql installations.
+
+Install the MySQL Binary: [https://dev.mysql.com/downloads/mysql/](https://dev.mysql.com/downloads/mysql/) 
+
+
+Download and follow setup instructions.
+
+**For MySQL 5.7 +**
+
+**After installing, you’ll need to do the following:**
+
+	$ touch /etc/my.cnf
+
+Add:
+
+
+	[mysqld]
+	sql_mode=STRICT_TRANS_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION
+
+[https://craftcookbook.net/recipes/making-craft-play-nicely-with-mysql-5-7-5](https://craftcookbook.net/recipes/making-craft-play-nicely-with-mysql-5-7-5) 
+
+### Install Sequel Pro
```

Comment:
I don't think we've committed to any specific DB editor across the board. I could see this going in the standard under software or in the laptop script by using `brew cask`